09/calculator_test.go

```diff
@@ -1,17 +1,57 @@
 package calculator_test
 
 import (
+	"calculator"
 	"testing"
 )
 
 func TestSubtract(t *testing.T) {
-	t.Fatal("Not implemented yet")
+	testCases := []struct {
+		a, b int
+		want int
+	}{
+		{a: 1, b: 1, want: 0},
+		{a: 3, b: 2, want: 1},
+		{a: 6, b: 7, want: -1},
```

Comment:
Another interesting test case might be if we subtract a negative number from a positive one.

Reply:
Thanks, added this test case.